### ISSUE TITLE
Setting default sorting

### ISSUE THREAD

**Issue by iproger** [magento/magento2#21493]
### Preconditions
1. Fresh Magento 2.3
2. Module Customer.

### Steps to reproduce
1. File: [Magento\Customer\Block\Adminhtml\Edit\Tab\Orders](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Customer/Block/Adminhtml/Edit/Tab/Orders.php)
2. Method: _construct().

### Expected result
```php
$this->setDefaultSort('created_at');
$this->setDefaultDir('desc');
```

### Actual result
```php
$this->setDefaultSort('created_at', 'desc');
```

Same problem in [Magento\Customer\Block\Adminhtml\Edit\Tab\View\Cart](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Customer/Block/Adminhtml/Edit/Tab/View/Cart.php).

**Comment by magento-engcom-team** [comment]
Hi @iproger. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@iproger do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-backlog-nazar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98482`, `MAGETWO-98483` were created

**Issue Available**: @engcom-backlog-nazar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @iproger. Thank you for your report. 
The issue has been fixed in magento/magento2#21498 by @Jitheesh in 2.3-develop branch
Related commit(s):
 - [50647c4458baaa14f6f48eed82cd4fb51a204a15](https://github.com/magento/magento2/commit/50647c4458baaa14f6f48eed82cd4fb51a204a15)

The fix will be available with the upcoming 2.3.2 release.

**Comment by magento-engcom-team** [comment]
Hi @iproger. Thank you for your report. 
The issue has been fixed in magento/magento2#21694 by @mage2pratik in 2.2-develop branch
Related commit(s):
 - [b4283aa17b03927d8f2b0983b9b519b684610b5e](https://github.com/magento/magento2/commit/b4283aa17b03927d8f2b0983b9b519b684610b5e)

The fix will be available with the upcoming 2.2.9 release.
